azkaban-web-server/src/web/js/azkaban/test/test_running_server.js

```diff
@@ -0,0 +1,42 @@
+/* After running `gradle clean installDist` this will get called by running `npm test` in the azkaban-web-server
+   directory.  These tests are used against a running Azkaban Solo server to confirm version changes to javascript
+   libraries don't break application web pages.
+
+   todo li-afaris: `gradle npm_test` will look at package.json and call this test, but the test fails due because mocha
+   can't be found.  It would be nice to fix the npm path when called by gradle.
+*/
+
+var chai   = require('chai');
+var chaiHttp = require('chai-http');
+var server = 'http://localhost:8081';
+var should = chai.should();
+
+chai.use(chaiHttp);
+
+// Test that Azkaban web interface is running
+describe("Will Azaban Login Work?", function() {
```

Comment:
Have limited knowledge of chai testing. Looks like this script can only test against HTTP status code? what does `version change` mean here? Is it able to catch any js or css library break?